packages/currency/integration-tests/__tests__/currency-module-service.spec.ts

```diff
@@ -0,0 +1,185 @@
+import { Modules } from "@medusajs/modules-sdk"
+import { ICurrencyModuleService } from "@medusajs/types"
+import { SqlEntityManager } from "@mikro-orm/postgresql"
+import { moduleIntegrationTestRunner, SuiteOptions } from "medusa-test-utils"
+import { Currency } from "../../src/models"
+
+jest.setTimeout(100000)
+
+moduleIntegrationTestRunner({
+  moduleName: Modules.CURRENCY,
+  testSuite: ({
+    MikroOrmWrapper,
+    service,
+  }: SuiteOptions<ICurrencyModuleService>) => {
+    describe("Currency Module Service", () => {
+      // TODO: Remove this and load currencies in loader
```

Comment:
This is the only todo that needs to be done before the PR is merged, but I decided to open it in a non-draft mode so others have the chance to review it in the meantime. The currencies will be loaded through a loader based on the final discussion we had with Adrien.